with-yarn-workspaces/apps/first-app/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "main": "__generated__/AppEntry.js",
+  "main": "index.js",
```

Comment:
why would this work, there is no index.js file

Reply:
`index.js` is generated by `expo-yarn-workspaces`. but yeah it should probably be committed -- anyway I'm closing this because this is not a good solution https://github.com/expo/expo/pull/13203#issuecomment-856957952